src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,68 @@
-import React from 'react';
 import { Loader } from '../Loader';
+import { useDispatch } from 'react-redux';
+import { currentTodoReducer } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
+import { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+export const TodoModal = ({ todo }: { todo: Todo }) => {
+  const dispatch = useDispatch();
+  const resetModal = () => dispatch(currentTodoReducer(null));
+
+  const [user, setUser] = useState({} as User);
```

Comment:
```suggestion
  const [user, setUser] = useState<User>(null);
```
better to set null as default value